README.md

```
@@ -310,6 +310,30 @@ We welcome contributions from developers, designers, and open-source enthusiasts
 - Pull request guidelines
 - Community communication
 
+## 🐳 Docker Support
+
+You can run OrgExplorer in a containerized environment using Docker or Docker Compose. This is useful for self-hosting in your own infrastructure.
+
+### Build and Run with Docker Compose (Recommended)
+
+1. Build and start the container:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)